src/components/client/landing/quotesSection/QuotesSection.jsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react'
+import "./quotesSection.css"
+
+import { Navigation, Pagination } from 'swiper'
+import { Swiper, SwiperSlide } from 'swiper/react'
+
+import 'swiper/css'
+import 'swiper/css/navigation'
+import 'swiper/css/pagination'
+import 'swiper/css/scrollbar'
+
+import quotes from './quotes.json'
+
+function QuotesSection() {
+  return (
+    <div className="container-fluid row px-2 pt-4 pb-4 quotesDiv">
+      <h1 className="text-center">Quotes</h1>
+      <Swiper modules={[Navigation, Pagination]} navigation pagination={{ clickable: true }}>
+        {
+          quotes.map((quote) => (
```

Comment:
First check whether quotes is existing or not

Reply:
on it!

Reply:
![Screenshot 2023-03-07 232332](https://user-images.githubusercontent.com/85297495/223507740-a00f68e0-0ae2-4f04-bc0b-ed1ba5fffcab.png)